packages/autorest.python/autorest/codegen/models/__init__.py

```diff
@@ -140,6 +140,7 @@
     "AzureKeyCredentialPolicy": AzureKeyCredentialPolicyType,
     "any-object": AnyObjectType,
     "unixtime": UnixTimeType,
+    "credential": StringType,
```

Comment:
is it possible if we hold off on this PR until laurent get's back? I'm not 100% sure what Python should do in this scenario. Thanks!